capstone/src/main/webapp/details.js

```diff
@@ -107,3 +110,19 @@ function appendCuisineTag(cuisineName) {
         .appendChild(outerDiv);
   }
 }
+
+/**
+ * Update the page views for a restaurant using the PageViewServlet.
+ * @param restaurantKey
+ * @param restaurantName
+ * @return {Promise<void>}
+ */
+async function updatePageViews(restaurantKey, restaurantName) {
+  const params =
+      'restaurantKey=' + restaurantKey + '&restaurantName=' + restaurantName;
```

Comment:
Is restaurant name actually needed here?

Reply:
We're including it so that I can label the lines by restaurant in my visualization!

Reply:
I understand including the restaurant name in the value returned by the servlet for the visualization, but would it be possible to not include it in the post for page views? It's usually nice to limit the source of truth to just one thing. For example, here if the restaurant key is for one restaurant and the restaurant name is for a different restaurant, which restaurant should the page views be incremented for? 

If restaurants each have a unique restaurant key, then that should be all the servlet needs to look up info/store info about a restaurant.

Reply:
Just leaving a drive by comment. We actually use the restaurantName for when we use the method getAllPageViews(). Having the name allows me to add the name of the restaurant to the return of the method, instead of having to return the key which would not serve of anything since the frontend wouldn't know who that key belongs to.

Reply:
The alternative would be me making a call to the restaurantClient to get the name of the restaurant's name. But we thought this would be easier.

Reply:
Removed the name! Thanks for the great discussion this morning